doc/source/changes.rst

```diff
@@ -14,6 +14,13 @@ Version 1.7.2 (CURRENT DEV)
 This is the current development version, not yet released.
 Changes thus far are relevant only for users of MapFish Print.
 
+Oereblex related changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Pyramid Oereb now support and use by default geoLink schema version 1.2.0:
+
+ * New doctype 'notice' (will be classed as 'HintRecord'). 'Notice' can have no authority nor enactment_date
+ * New document attribute 'language' and file attribute 'description' are currently not used by pyramid_oereb.
```

Comment:
should we add here something like: "but are now available to custom code, for example for document title generation."

(several cantons have custom code regarding this)